app/components/check_your_answers_summary_component.rb

```diff
@@ -52,5 +102,15 @@ def format_value(value, opts)
     value.to_s
   end
 
+  def file_names_for(document, translations)
+    scope =
+      translations ? document.translated_uploads : document.original_uploads
+    scope
+      .order(:created_at)
+      .map { |upload| upload.attachment&.filename }
+      .compact
+      .join(", ")
+  end
```

Comment:
```suggestion
  def file_names_for(document, translation)
    document.uploads
      .where(translation:)
      .order(:created_at)
      .filter_map { |upload| upload.attachment&.filename }
      .join(", ")
  end
```

Just shortens the code a little, but I don't mind either way.

I also wonder if we should copy the array field and have `.join("<br />").html_safe` but that's out of scope.

Reply:
I think I'll leave this one as `.translated_uploads` and `.original_uploads` encapsulate the definition of 'a translation' in the document model rather than bleeding the database field into the view component. (we've arguably done that anyway with `created_at` 🤷)